charts/fission-all/values.yaml

```diff
@@ -267,8 +267,10 @@ analytics: true
 ## Internally used for generating an analytics job for non-helm installs
 analyticsNonHelmInstall: false
 
-## Enable Heapster only in clusters where heapster does not exist already
-heapster: false
+# ## Enable Heapster only in clusters where heapster does not exist already
+# heapster: false
```

Comment:
From the above file(templates/deployment.yaml), I noticed heapster was removed. Does this need to be removed too?